Sources/KlaviyoSwift/AppContextInfo.swift

```diff
@@ -20,10 +20,14 @@ struct AppContextInfo {
     private static let defaultOSName = "iOS"
     private static let defaultDeviceModel: String = {
         var size = 0
+        var deviceModel = ""
```

Comment:
does our backend support empty string values for device model?

Reply:
Good question. I tried sending a empty string and it seems to accept w/o any issue. May be instead of empty string we can have it as unknown?
```json
{
  "SDK Version": "2.3.0",
  "App Version": "1.0",
  "SDK Name": "swift",
  "OS Version": "17.1.1",
  "Total Price": 10.99,
  "Push Token": null,
  "App ID": "com.klaviyo.testiosforbfcm",
  "Items Purchased": [
    "Hot Dog",
    "Fries",
    "Shake"
  ],
  "App Build": "1",
  "App Name": "testingKlaviyo",
  "Device Manufacturer": "Apple",
  "Device ID": "XXXXX",
  "Device Model": "",
  "OS Name": "iOS",
  "$value": 10.99
}
```

Reply:
Probably best if it is empty string. If the model or manufacturer are empty, the device metadata service will just skip those, with no effect on other metadata fields. If you sent "Unknown" then we'd have DeviceModel records specifically named "Unknown"